js-client/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "declaration": true,
+    "declarationMap": true,
+    "emitDecoratorMetadata": true,
+    "esModuleInterop": true,
+    "experimentalDecorators": true,
+    "forceConsistentCasingInFileNames": true,
+    "module": "esnext",
+    "lib": ["dom", "esnext"],
```

Comment:
For contracts and subgraph, we use

```
    "lib": ["es6"],
    "module": "commonjs",
```
Should we unify all of them? What is the better way?

Reply:
This will happen in https://aragonassociation.atlassian.net/browse/OS-825 so we can close this conversation